src/App.tsx

```diff
@@ -1,26 +1,235 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import {
+  createTodo,
+  deleteTodo,
+  editTodo,
+  getTodos,
+  USER_ID,
+} from './api/todos';
+import { FilterOptions, Todo } from './types';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { getPrepearedTodos } from './utils/getPrepearedTodos';
+import { Footer } from './components/Footer';
+import { Error as ErrorNotification } from './components/Error';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterCompleting, setFilterCompleting] = useState<FilterOptions>(
```

Comment:
```suggestion
  const [filterType, setFilterCompleting] = useState<FilterOptions>(
```

Reply:
So what's wrong with this naming? It's much more clear than just a 'type'

Reply:
This is not a very correct name, for example, it is not very clear to me what lies in filterCompleting. It looks more like filtering is complete. And so filterType means that here is the filtering type.